x/txfees/keeper/hooks_test.go

```diff
@@ -104,6 +104,11 @@ func (s *KeeperTestSuite) TestTxFeesAfterEpochEnd() {
 				s.NoError(err)
 				err = s.App.BankKeeper.SendCoinsFromAccountToModule(s.Ctx, addr0, types.FeeCollectorForStakingRewardsName, sdk.Coins{coin})
 				s.NoError(err)
+
+				// Update the tx fee tracker
+				currentTxFeesTrackerValue := s.App.TxFeesKeeper.GetTxFeesTrackerValue(s.Ctx)
+				newTxFeesTrackerValue := currentTxFeesTrackerValue.Add(sdk.NewCoins(coin)...)
+				s.App.TxFeesKeeper.SetTxFeesTrackerValue(s.Ctx, newTxFeesTrackerValue)
```

Comment:
Was there a reason you added this here? Maybe the intention was to call GetTxFeesTrackerValue afterwards ?-?

Reply:
Ah thanks for pointing out, before I was updating the trackers at epoch during the swap, but realized it isn't necessary and removed it. Will remove this as well.

Reply:
Removed https://github.com/osmosis-labs/osmosis/pull/6804/commits/bc3ddee9c75b3384d42bee5165ae2bc63b1abf4b